scss/bootstrap-reboot.scss

```diff
@@ -12,4 +12,5 @@
 $font-family-base: $font-family-sans-serif; // stylelint-disable-line scss/dollar-variable-default
 $font-family-code: $font-family-monospace; // stylelint-disable-line scss/dollar-variable-default
```

Comment:
Are those still useful then? They did the exact same job.

Moreover I think we now import tons of unused custom properties in reboot. If going this way, I'd recommend to only duplicate what's needed, rather than importing the whole file.

Does this make sense?

Reply:
Yeah, we need to clean that up before merging. I don't know why or when we decided no `:root` vars in Reboot. Definitely has been the direction I've wanted to go all along haha. I do understand the concern about bloat though. Will think on it.